### PR TITLE
Record Edit Post Submission Page

### PR DESCRIPTION
**Pull request by RFSH**
With the new changes in ermerstjs, now create and edit will return a object that contains successful and failure. so chaise doesn't need to check for the failure cases and can show these two pages.
Also the returned page is contextualize and chaise doesn't need to contextualize it again.

This PR also changes the `bind-html-unbind` that has been used in multiple apps to `ng-bind-html`.

### CODE REVIEW

File: recordedit/form.controller.js [hunk 1]
```diff
@@ -282,10 +282,11 @@
 
                     if (model.rows.length == 1) {
                         vm.redirectAfterSubmission(page);
-                    } else {
+                    } else { // mutli create/edit: show result
```
Comment (jrchudy) [inline recordedit/form.controller.js]: typo

File: recordedit/form.controller.js [hunk 2]
```diff
@@ -252,16 +252,16 @@
...
+                    // the returned reference is contextualized and we don't need to contextualize it again
                     var resultsReference = page.reference;
 
                     if (isUpdate) {
-                        resultsReference = $rootScope.reference.contextualize.compact;
-                        // check if there is a window that opened the current one
-                        // make sure the update function is defined for that window
-                        // verify whether we still have a valid vaue to call that function with
-                        if (window.opener && window.opener.updated && context.queryParams.invalidate) {
+                        if (window.opener && window.opener.updated) {
```
Comment (jrchudy) [inline recordedit/form.controller.js]: Looks like you overwrote this change that I made yesterday to address the `window.update` issue. These changes are from PR #1146.

Comment (RFSH) [inline recordedit/form.controller.js]: My master branch when I started working was older than yours. I just merged it with master. That's why you see these changes.

**Comment by RFSH** [comment]
@jrchudy I made the changes and addressed your comments.

**Comment by jrchudy** [comment]
I verified that the tests pass locally from start to finish for me using your `ermrestJS` branch and `Chaise` branch for this issue.

**Pull request merged**